config/api.go

```diff
@@ -40,5 +42,7 @@ func setAPIDefault() {
 	viper.SetDefault(ServicePathDocker, filepath.Join("/mesg", "services"))
 	os.MkdirAll(viper.GetString(ServicePathDocker), os.ModePerm)
 
-	viper.SetDefault(CoreImage, "mesg/core:latest")
+	// Keep only the first part if Version contains space
+	coreTag := strings.Split(version.Version, " ")
+	viper.SetDefault(CoreImage, "mesg/core:"+coreTag[0])
```

Comment:
I fix a bug here when the dev release, available on GitHub, start the image with `mesg/core:dev build XXXXX`.
Now it start with image `mesg/core:dev`.